cert/lib/cert/runner.rb

```diff
@@ -123,9 +123,17 @@ def certificates
 
     # The kind of certificate we're interested in
     def certificate_type
-      cert_type = Spaceship.certificate.production
-      cert_type = Spaceship.certificate.in_house if Spaceship.client.in_house?
-      cert_type = Spaceship.certificate.development if Cert.config[:development]
+      case Cert.config[:platform].to_s
+      when 'ios', 'tvos'
```

Comment:
This is for developer signing certificates only.  As far I know `tvos` isn't a valid type for developer signing.

FYI, for `sigh` (aka provisioning profiles) both `ios` and `tvos` are supported; as well as `macos`.